.pylintrc

```diff
@@ -315,13 +315,6 @@ max-line-length=100
 # Maximum number of lines in a module.
 max-module-lines=1000
 
-# List of optional constructs for which whitespace checking is disabled. `dict-
-# separator` is used to allow tabulation in dicts, etc.: {1  : 1,\n222: 2}.
-# `trailing-comma` allows a space between comma and closing bracket: (a, ).
-# `empty-line` allows space-only lines.
-no-space-check=trailing-comma,
```

Comment:
This option was removed in [Pylint 2.6.0](https://github.com/PyCQA/pylint/releases/tag/pylint-2.6.0).